CHANGELOG.md

```diff
@@ -1,3 +1,107 @@
+# 1.1.0 (2019-12-04)
+
+It's been a busy 2 weeks since the 1.0.0 release and quite a few important PR's have been merged to Loki.
+
+The most significant:
+
+* [1322](https://github.com/grafana/loki/pull/1322) **rfratto**: Fix v1 label API to be Prometheus-compatible
+
+Some might call this a **breaking change**, we are instead calling it a bug fix as our goal was to be prometheus compatible and we were not :smiley:  
+
+**But please be aware if you are using the `/loki/api/v1/label` or `/loki/api/v1/label/<name>/values` the JSON result will be different in 1.1.0**
+
+Old result:
+```json
+{
+  "values": [
+    "label1",
+    "label2",
+    "labeln"
+  ]
+}
+```
+New result:
+
+```json
+{
+  "status": "success",
+  "data": [
+    "label1",
+    "label2",
+    "labeln"
+  ]
+}
+``` 
+
+**ALSO IMPORTANT**
+
+* [1160](https://github.com/grafana/loki/pull/1160) **daixiang0**: replace zip with gzip
```

Comment:
Even though that's the real title of the PR, it confused me and makes it sound like we're moving towards gzip. Maybe we should refer to it as "replace gzip with zip" here?

Reply:
ah yeah, i'll tweak that, it is backwards